Kwartracker/Custom Components/TextField/BirthdateField+View.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  TextField+Extension.swift
+//  Kwartracker
+//
+//  Created by Mickale Saturre on 4/29/21.
+//
+
+import SwiftUI
+
+struct BirthdateField: View {
+    var textField: TextField<Text>
+    
+    private let colorOpacity: Double = 0.3
+    private let calendarImageSize: CGFloat = 15
+    private let leadingSideMargin: CGFloat = 20
+    private let calendarContainerSize: CGFloat = 40
+    private let calendarCotnainerRadius: CGFloat = 13
+
+    var body: some View {
+        HStack {
+            textField
```

Comment:
instead of receiving view, why not just make this custom and just make the values dynamic